docs/guides/add-new-provider.md

```diff
@@ -65,12 +65,19 @@ Provider name serves primarily for identification. We recommend it to be as shor
 The easiest way to bootstrap a new provider is using [Superface CLI](/reference/cli).
 
 ```shell
-superface create --provider --providerName <provider-name>
+superface prepare:provider <provider-name>
 ```
 
 _Replace the `<provider-name>` in the command with the actual name you wish to use._
 
-Running the above command creates a new JSON file at `<provider-name>.provider.json` and links the provider in the local `super.json` configuration file.
+Running the above command interactively guides you through creation of new provider JSON file at `<provider-name>.provider.json` and links the provider in the local `super.json` configuration file.
```

Comment:
```suggestion
Running the above command interactively guides you through the creation of a new provider JSON file at `<provider-name>.provider.json`, and links the provider in the local [`super.json` configuration file](/docs/advanced-usage).
```

 I'd like to eventually make a separate `super.json` page, for now it lives on `advanced-usage`